src/less/tooltip/ds6/tooltip.less

```diff
@@ -0,0 +1,205 @@
+@import "../../less/ds6/variables.less";
+@import "../../less/ds6/mixins.less";
+
+@import "tooltip-mixins.less";
+@import "../base/tooltip.less";
+
+.tooltip {
+    &--tooltip,
+    &--tourtip {
+        background-color: @ds6-color-p002-blue;
+        color: @ds6-color-g201-gray;
+    }
+
+    &--infotip {
+        background-color: @ds6-color-g201-gray;
+        box-shadow: 0 0 3px 1px @ds6-color-g205-gray;
+        color: @ds6-color-g206-gray;
+
+        .infotip {
+            &__arrow-top {
+                &-left,
+                &-center,
+                &-right {
+                    box-shadow: -2px -2px 2px @ds6-color-g205-gray;
+                }
+            }
+
+            &__arrow-bottom {
+                &-left,
+                &-center,
+                &-right {
+                    box-shadow: 2px 2px 2px @ds6-color-g205-gray;
+                }
+            }
+
+            &__arrow-left {
+                &-top,
+                &-center,
+                &-bottom {
+                    box-shadow: -2px 2px 2px @ds6-color-g205-gray;
+                }
+            }
+
+            &__arrow-right {
+                &-top,
+                &-center,
+                &-bottom {
+                    box-shadow: 2px -2px 2px @ds6-color-g205-gray;
+                }
+            }
+        }
+    }
+
+    &--tooltip,
+    &--infotip,
+    &--tourtip {
+        font-size: @ds6-font-size-14;
+        padding: 16px;
+        position: absolute;
+        z-index: 1;
+
+        .tooltip,
+        .infotip,
+        .tourtip {
+            &__arrow-top-left {
+                .tooltip-arrow-top;
+
+                left: 12px;
+            }
+
+            &__arrow-top-center {
+                .tooltip-arrow-top;
+
+                left: calc(50% - 8px);
+            }
+
+            &__arrow-top-right {
+                .tooltip-arrow-top;
+
+                right: 12px;
```

Comment:
We should apply `12px` as a variable, then it can be applied and updated consistently.  (For arrow-bottom-left it is 16px right now instead of 12 like the others)